deploy/deploy.go

```diff
@@ -14,11 +14,12 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Deployment check timeout in minutes
-const timeoutMins = 5
-
-// Check interval in seconds
-const checkIntervalSecs = 15
+var (
```

Comment:
I feel like these should be in the config, and then passed to the deploy land as arguments after being parsed in main. What do you think?